libs/books/data-access/src/lib/+state/reading-list.reducer.ts

```diff
@@ -70,6 +70,9 @@ const readingListReducer = createReducer(
       { bookId: action.item.bookId, ...action.item },
       state
     )
+  ),
+  on(ReadingListActions.markedAsFinished, (state, action) =>
+    readingListAdapter.updateOne({ id: action.item.id, ...action.item }, state)
```

Comment:
Change in reducer to Undo adding list, Undo Remove reading list and mark as finished

Reply:
Is it updating book details with `finished` and `finishedDate` property?